changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.5.0-beta3": [
+      "[???] Add support for reporting a binary file is in a conflicted state - #6072",
+      "[???] Handle really long banner messages - #6050",
+      "[Fixed] Ensure status after aborting a merge reflects reality - #6052",
```

Comment:
The three fixes here are part of the Conflict Resolution flow that users noticed on beta, but we should omit it from the production release notes :fire: